piksi_tools/console/settings.yaml

```diff
@@ -2199,3 +2199,202 @@
   Description: (ALPHA) Maximum possible linear acceleration of vehicle.
   Notes: Has no effect unless the vehicle dynamics filter is active.
 
+- group: ins
+  name: output_mode
+  expert: false
+  type: enum
+  units: N/A
+  default value: 'disabled'
+  readonly: false
+  enumerated possible values: Disabled, Loosely Coupled, LC + GNSS
+  Description: Determines output mode of the inertial navigation outputs. 
+  Notes: Disabled configures device to output GNSS only solutions.
```

Comment:
Suggested comment:
```Disabled will disable the inertial solution - device is configured to output GNSS only solutions 
Loosely Coupled will output a loosely coupled GNSS and inertial solution
LC + GNSS configures a testing mode that simultaneously outputs both GNSS only and the loosely coupled inertial solution.
In LC + GNSS mode, there are two sets of solution messages with identical times. The flags indicate which messages are GNSS only solutions and which include are the loosely coupled inertial solution```

Reply:
I will remove double spaces.  I will remove all abbreviations of INS in the descriptions.